composer.json

```diff
@@ -21,7 +21,7 @@
         "php": "^7.4 || ^8.0",
         "ext-json": "*",
         "qase/api": "v1.0.1",
-        "qase/php-client-utils": "v1.0.0",
+        "qase/php-client-utils": "v1.0.1",
```

Comment:
@sklmx I am changing here to v1.0.1 for now, but I think it is better to have `"^1.0"` like in PHPUnit reporter: It will update the patch version to the latest on every `"composer update"`.

Reply:
Sorry. Changed to "^1.0.1"